content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/differences.rst

```diff
@@ -2,46 +2,94 @@
 What's the difference between lots and serial numbers?
 ======================================================
 
-Introduction
-============
+*Lots* and *serial numbers* are the two ways to identify and track products in Odoo. While there are
+similarities between the two traceability methods, there are also notable differences that affect
+receipts, deliveries, and inventory reports.
+
+A lot usually indicates a specific batch of an item that was received, is currently stored, or was
+shipped from a warehouse, but can also pertain to a batch of products manufactured in-house, as
+well.
```

Comment:
```suggestion
A *lot* usually indicates a specific batch of an item that was received, is currently stored, or was
shipped from a warehouse. However, it can also pertain to a batch of products manufactured in-house,
as well.
```